lib/persisted-model.js

```diff
@@ -897,9 +897,10 @@ module.exports = function(registry) {
         description: 'Get a set of deltas and conflicts since the given checkpoint.',
         accessType: 'READ',
         accepts: [
-          {arg: 'since', type: 'number', description: 'Find deltas since this checkpoint'},
-          {arg: 'remoteChanges', type: 'array', description: 'an array of change objects',
-            http: {source: 'body'}},
+          { arg: 'since', type: 'number', description: 'Find deltas since this checkpoint',
+            http: { source: 'form' }},
```

Comment:
Thanks for following up @bajtos. I'm a big fan. I've been working with loopback for a few years. After reading so much of your work and posts, actually interacting with you is like talking to a rock star.  :)

It's not clear to me why changing the source for the input parameters is a breaking change. I assumed the method that gets called was agnostic to the source of the parameters and that strong-remoting would handle mapping the parameters appropriately. But maybe thats not the case. Or maybe there is another place in the framework that depends on remoteChanges being mapped to body?

Whatever the case I'm happy to go with your suggestion. I updated the pull request with what I think you are suggesting. The code looks for an option "explicitSinceSource" I signed the CLA and made sure npm test passes. 

I did not add a test to catch the error when diff is called via rest. I am not sure how I would write a test that examines the parameters before they are sent to the server. If you can point me to an example you think demonstrates how I could do this I'd be happy to review and take a shot at it.